docs/API-Reference/bpmn-js/importXML.md

```diff
@@ -10,3 +10,62 @@ grand_parent: "API Reference"
 ---
 
 # importXML
+
+`importXML` API parses and renders a BPMN 2.0 diagram.
+
+## Usage
+
+importXML(xml: _[String](#)_, bpmnDiagram: _[ModdleElement](#)_ or _[String](#)_): Promise<_[ImportXMLResult](#importxmlresult)_, _[ImportXMLError](#importxmlerror)_>
+
+* `xml`: The BPMN 2.0 XML.
+* `bpmnDiagram`: The BPMN diagram or the id of the diagram to render _(if not provided, the first one will be rendered)_.
+
+### Code Example
+
+```javascript
+const modeler = new BpmnJS();
+
+try {
+
+  const result = await modeler.importXML(xml);
```

Comment:
I'd love the example to be runnable. With top-level `await`, it's not yet available.

Reply:
What about switching to `.then(...)`?

Reply:
Yeah I don't think the code could be runnable without specifiying the DOM element and import statements. It does not make sense to include those in every code snippet IMO. I also removed the modeler construction code and added a comment instead. In this context, await is fine here IMO. It makes it clear that this API is awaitable / returns a Promise.

Reply:
We cannot include all the setup code in every example so we shouldn't try. We should rather decide on the naming that we use throughout the docs. Are we going to use `viewer` and `modeler`? Or `bpmnJS`?

Reply:
:+1:

Reply:
I updated the comment with: `modeler is an instance of BpmnJS Modeler.` So at that point it's clear that we're using a Modeler for given example. Wheter one should use a modeler or a viewer should become clear via other pages such as `Getting Started` etc. It's not the concern of the importXML API reference page.